README.md

```diff
@@ -1,72 +1,224 @@
+
+# Reservify app
+![Rails](https://img.shields.io/badge/rails-%23CC0000.svg?style=for-the-badge&logo=ruby-on-rails&logoColor=white)
+![Postgres](https://img.shields.io/badge/postgres-%23316192.svg?style=for-the-badge&logo=postgresql&logoColor=white)
+![Bootstrap](https://img.shields.io/badge/bootstrap-%23563D7C.svg?style=for-the-badge&logo=bootstrap&logoColor=white)
+![React](https://img.shields.io/badge/react-%23316192.svg?style=for-the-badge&logo=react&logoColor=white)
+![Redux](https://img.shields.io/badge/redux-%23563D7C.svg?style=for-the-badge&logo=redux&logoColor=white)
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](./MIT.md)
+![rails version](https://img.shields.io/badge/Rails-7-red)
+![ruby version](https://img.shields.io/badge/Ruby-3.x-orange)
+
+## Table of Contents
+- [Api Documentation](#api-documentation)
+- [Description](#description)
+- [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Clone Repo](#clone-repo)
+  - [Prerequisites](#prerequisites)
+  - [Install Rails](#install-rails)
+- [Setup](#setup)
+  - [System Dependencies](#system-dependencies)
+  - [Database Initialization](#database-initialization)
+- [Usage](#usage)
+  - [Start Server](#start-server)
+  - [Testing RSpec](#testing-rspec)
+- [How To Contribute?](#how-to-contribute)
+- [Collaborators](#collaborators)
+- [Show Your Support](#how-your-support)
+- [Acknowledgments](#acknowledgments)
+- [License](#license)
+<hr>
+
+## Api Documentation
+>You can review the documentation for the API [here](https://documenter.getpostman.com/view/13085631/Uz5MEZ6D#d4697ce8-cda6-4658-9891-aca869e0acc9).
+<hr>
+
+## Description
+> Web application where users should sign in once you are in you can choose a hotel from a list then reserve, cancel and see your room reservation.
+<hr>
 
-# Project Name
+## Built With
 
-> Description the project.
+- Major languages: Ruby, HTML, Javascript, Bootstrap 5.
+- Frameworks: Ruby on Rails, React&Redux.
+- Tools & Methods: RSpec, PostgreSQL, JWT, Jest, Sass, Linters.
+<hr>
 
+## Getting Started
 
-## Built With
+We'll need to prepare your computer to handle Ruby on Rails, which might be a hassle, so take a deep breath and cope with me :)
 
-- Major languages
-- Frameworks
-- Technologies used
+If you have Rails 7 installed, you can skip to the [Setup](#setup) section
 
-## Live Demo (if available)
+## Clone Repo
 
-[Live Demo Link](https://livedemo.com)
+First clone this repo by running this command in your terminal:
+~~~ bash
+git clone https://github.com/Yothu/reservify-back-end.git
+# wait a couple seconds for it to download
+# ...
+# ...
+# then move into the new directory
+cd reservify-back-end
+~~~
 
+<hr>
 
-## Getting Started
+## Prerequisites
+
+The following technologies must be installed before hand in your local machine:
+
+ - Ruby (ruby --version). Should be 3.0.1 or newer.
+ - SQLite3 (sqlite3 --version)
+ - Node.js (node --version)
+ - Yarn (yarn --version)  
```

Comment:
- Maybe I'm wrong but, I'm gonna ask anyway. Why use yarn on the back-end when the front is in another project? Maybe there's some detail I'm not considering, but if the Front-end is on another part, maybe using JS on this part is not required. 🐸

Reply:
### Thanks, @vichuge for the review!
We made the changes you advised us to do, but we have a problem, we cannot find a way to fix the vulnerability problems when using the `npm install` command.

It would be much appreciated if you could give us some advice on the matter.

Thanks!

Reply:
Well, it looks like your project is approved so, congrats!! 🎉